.dotnet/src/Custom/Audio/AudioTranscription.cs

```diff
@@ -7,19 +7,19 @@ namespace OpenAI.Audio;
 
 public partial class AudioTranscription
 {
-    public string Language { get; }
+    public string? Language { get; }
     public TimeSpan? Duration { get; }
     public string Text { get; }
     public IReadOnlyList<TranscribedWord> Words { get; }
-    public IReadOnlyList<TranscriptionSegment> Segments { get; }
+    public IReadOnlyList<TranscribedSegment> Segments { get; }
 
-    internal AudioTranscription(string language, TimeSpan? duration, string text, IReadOnlyList<TranscribedWord> words, IReadOnlyList<TranscriptionSegment> segments)
+    internal AudioTranscription(string? language, TimeSpan? duration, string? text, IReadOnlyList<TranscribedWord>? words, IReadOnlyList<TranscribedSegment>? segments)
     {
         Language = language;
         Duration = duration;
-        Text = text;
-        Words = words;
-        Segments = segments;
+        Text = text!;
```

Comment:
Should Text be declared as nullable here?  Why use the null-forgiving operator (`!`)?